test/smoketest/requirements.in

```diff
@@ -0,0 +1,8 @@
+# "Abstract" dependencies for the project
+#
+# The sibling requirements.txt file contains the corresponding concrete /
+# pinned dependencies.  You can use pip-tools to automatically update
+# requirements.txt from this file:
+# https://github.com/jazzband/pip-tools
```

Comment:
Okay, I confirmed another option to updating requirements.txt is to run the following after creating a fresh virtualenv:

    pip install -r requirements.in
    pip freeze > requirements.txt